.github/workflows/test.yml

```diff
@@ -33,8 +36,6 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-mix-${{ matrix.otp }}-${{ matrix.elixir }}-${{ hashFiles('**/mix.lock') }}
           ${{ runner.os }}-mix-${{ matrix.otp }}-${{ matrix.elixir }}
-          ${{ runner.os }}-mix-${{ matrix.otp }}
-          ${{ runner.os }}-mix
```

Comment:
Why were these keys dropped? The change itself is reasonable but the context behind your decision is missing.

Reply:
Well, you already described just the same changes in https://github.com/duffelhq/paginator/pull/104

Reply:
Sure, it's still helpful to be explicit about the reasons for making a change. Thank you for articulating it!